SF iOS/SF iOS/AppDelegate.m

```diff
@@ -7,57 +7,34 @@
 //
 
 #import "AppDelegate.h"
-#import "EventDataSource.h"
-#import "EventsFeedViewController.h"
 #import "NSNotification+ApplicationEventNotifications.h"
+#import "SwipableNavigationContainer.h"
 
 @interface AppDelegate ()
 
-@property (nonatomic) EventDataSource *dataSource;
+@property (nonatomic) SwipableNavigationContainer *navigationContainer;
 
 @end
 
 @implementation AppDelegate
 
-- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
-    self.window = [[UIWindow alloc] initWithFrame:[[UIScreen mainScreen] bounds]];
-    [self.window makeKeyAndVisible];
-    
-    EventDataSource *datasource = [[EventDataSource alloc] initWithEventType:EventTypeSFCoffee];
-    EventsFeedViewController *feedController = [[EventsFeedViewController alloc] initWithDataSource:datasource];
-    
-    self.window.rootViewController = feedController;
-    
-    return true;
-}
-
-
-- (void)applicationWillResignActive:(UIApplication *)application {
-    // Sent when the application is about to move from active to inactive state. This can occur for certain types of temporary interruptions (such as an incoming phone call or SMS message) or when the user quits the application and it begins the transition to the background state.
-    // Use this method to pause ongoing tasks, disable timers, and invalidate graphics rendering callbacks. Games should use this method to pause the game.
-}
-
-
-- (void)applicationDidEnterBackground:(UIApplication *)application {
-    // Use this method to release shared resources, save user data, invalidate timers, and store enough application state information to restore your application to its current state in case it is terminated later.
-    // If your application supports background execution, this method is called instead of applicationWillTerminate: when the user quits.
+- (BOOL)application:(UIApplication *)application willFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    self.navigationContainer = [[SwipableNavigationContainer alloc] init];
```

Comment:
Oh you made it a nav! Now I gotta build and run

Reply:
Design feedback. I like how big the text is. I'm wondering if there is a scenario where we could change it to a scrollview or more specifically a collectionview at some point. I love keeping the context of: "Oh yes, the feed is there on the right" with a little bit of the feed showing on the right. It's a much later solution. RN I'm down to put this PR into master.